test/framework/easyconfigs/test_ecs/t/toy/toy-0.0-gompi-1.3.12-test-description.eb

```diff
@@ -0,0 +1,34 @@
+name = 'toy'
+version = '0.0'
+versionsuffix = '-test'
+
+homepage = 'http://hpcugent.github.com/easybuild'
+description = "Toy C program. Now with `backticks'"
```

Comment:
rather than adding a new easyconfig, can we just use `toy-0.0.eb`, read it, do:

```python
toytxt = re.sub('description = .*', 'description = "Toy C program. Now with `backticks'", toytxt)
```

and then write it to a temporary file to use for the test?